Utilities/Debugger/ITK.natvis

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!-- Documentation: https://msdn.microsoft.com/en-us/library/jj620914.aspx?f=255&MSPPError=-2147217396 -->
 <!-- e.g. copy into C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Packages\Debugger\Visualizers\ITK.natvis -->
-<AutoVisualizer xmlns="https://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
```

Comment:
I guess this fix should also be applied to the XML file "default.natstepfilter": 

https://github.com/InsightSoftwareConsortium/ITK/blob/3bbcb195fb3f8ed84d7e2081eced74ff1b0baf32/Utilities/Debugger/default.natstepfilter#L3 

But I don't really know yet. @Leengit Dit you intend to _also_ adjust those XML namespace names to use "https", when you submitted PR #3428 ?

Reply:
Update: officially, I think that the ".natstepfilter" file should just have "http", not "https", in the `xmlns` attribute of `StepFilter`. Looking at _Learn / Visual Studio / Debugging_ section [Customize C++ stepping behavior independent of Just My Code settings](https://learn.microsoft.com/en-us/visualstudio/debugger/just-my-code?view=vs-2022#BKMK_CPP_Customize_stepping_behavior):

> A _.natstepfilter_ file is an XML file with this syntax:
```

<?xml version="1.0" encoding="utf-8"?>
<StepFilter xmlns="http://schemas.microsoft.com/vstudio/debugger/natstepfilter/2010">
    <Function>
        <Name>FunctionSpec</Name>
        <Action>StepAction</Action>
    </Function>
    <Function>
        <Name>FunctionSpec</Name>
        <Module>ModuleSpec</Module>
        <Action>StepAction</Action>
    </Function>
</StepFilter>

``` 

However, it seems to me the Visual Studio debugger entirely ignores the xmlns attribute of `StepFilter` (as I just observed, using VS2022), so it may not matter so much anyway.  🤷  Still, for a _.natvis_ file, an incorrect xmlns attribute is a show-stopper.

Reply:
> But I don't really know yet. @Leengit Dit you intend to _also_ adjust those XML namespace names to use "https", when you submitted PR #3428 ?

I didn't differentiate contexts or file types.  Any http-style URL that worked via curl as a https-style URL was changed.  A few of those changes were manually undone: if a change proved to be a test breaker and couldn't be fixed by hand in some other way then it was undone.